niworkflows/viz/utils.py

```diff
@@ -367,7 +367,7 @@ def plot_registration(
             display.add_contours(white, colors="b", **kwargs)
             display.add_contours(pial, colors="r", **kwargs)
         elif contour is not None:
-            display.add_contours(contour, colors="b", levels=[0.5], linewidths=0.5)
+            display.add_contours(contour, colors="r", levels=[0.5], linewidths=0.5)
```

Comment:
this will clobber the pial (line 368), can we find some better color?

Reply:
those should be mutually exclusive, since it's an `if elif` block no?